bench/wtperf/wtperf.c

```diff
@@ -1420,16 +1651,19 @@ execute_workload(CONFIG *cfg)
 {
 	CONFIG_THREAD *threads;
 	WORKLOAD *workp;
-	uint64_t last_ckpts, last_inserts, last_reads, last_updates;
+	uint64_t last_ckpts, last_inserts, last_reads, last_truncates;
+	uint64_t last_updates;
 	uint32_t interval, run_ops, run_time;
 	u_int i;
 	int ret, t_ret;
 	void *(*pfunc)(void *);
 
 	cfg->insert_key = 0;
-	cfg->insert_ops = cfg->read_ops = cfg->update_ops = 0;
+	cfg->insert_ops = cfg->read_ops =
+	    cfg->truncate_ops = cfg->update_ops = 0;
```

Comment:
We don't split lines for local variable initialization, this would become:

```
cfg->insert_ops = cfg->read_ops = cfg->update_ops = 0;
cfg->update_ops = 0;
```